invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/records/search/SearchResultItemLayout.js

```diff
@@ -57,7 +58,7 @@ class SearchResultItemComponent extends Component {
             icon="lock"
           />
           <a target="_blank" rel="noreferrer" href={result.links.self_html}>
-            {_truncate(result.metadata.title, { length: 50 })}
+            {_truncate(result.metadata.title || i18next.t("Untitled"), { length: 50 })}
```

Comment:
nit: in case we want to be more explicit
```suggestion
            {_truncate(result.metadata.title || i18next.t("Empty draft title"), { length: 50 })}
```

Reply:
Thanks to my background in VFX, the default name for an unnamed layer is "Untitled". However, if you prefer replacing one word with three for admins, we can go ahead with that.